web-client/public/js/grnstate.js

```diff
@@ -0,0 +1,3 @@
+export var grnState = {
+    GRAVITY_LENGTH_WITHOUT_ZERO = 3
+}
```

Comment:
The presence of the object itself is in the right direction, but I saw it more as a container for values that will change, and not constants. If you declare `GRAVITY_LENGTH_WITHOUT_ZERO` in this way, it is still possible to change it.

Better to really declare it as a constant (e.g., `const GRAVITY_LENGTH_WITHOUT_ZERO = 3` or as a property that is _read-only_). Instead, try modeling a value here that _will_ change while the app is running.